src/pages/overview/pdf-electronic-seal-api/quickstarts.md

```diff
@@ -40,19 +40,19 @@ The seal field parameters are required to create a new unsigned signature field
 * **Field Name**  (*Required*): The signature field's name. This must be a non-empty string. If signature field with this field name already exist, that field will be used. 
```

Comment:
The seal field parameters are required to create a new signature field or use an existing field.

Reply:
Done